search-improvements.rst

```diff
@@ -0,0 +1,51 @@
+.. post:: August 14, 2018
+   :tags: gsoc, search, feature
+   :author: Safwan
+   :location: DHA
```

Comment:
To make the location work, you need an entry here: https://github.com/rtfd/blog/blob/master/conf.py#L54